sdks/wasm/DebuggerTestSuite/Tests.cs

```diff
@@ -123,6 +123,70 @@ public class SourceList : DebuggerTestBase {
 			});
 		}
 
+		[Fact]
+		public async Task CheckBreakpoint () {
+			var insp = new Inspector ();
+
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready ();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+
+				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 44);
+				Assert.EndsWith ("debugger-array-test.cs", bp1_res.Value["breakpointId"].ToString());
+				Assert.Equal (1, bp1_res.Value ["locations"]?.Value<JArray> ()?.Count);
+
+				var loc = bp1_res.Value ["locations"]?.Value<JArray> ()[0];
+
+				Assert.NotNull (loc ["scriptId"]);
```

Comment:
You could use `CheckLocation ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 44)`.